packages/sources/alpine/README.md

```diff
@@ -0,0 +1,87 @@
+# Chainlink External Adapter for Alpine
+
+This adapter gets the tvl of Ethereum vaults as well as the block numbers of the last cross chain transfers.
+
+### Environment Variables
+
+| Required? |  Name   |         Description          | Options | Defaults to |
+| :-------: | :-----: | :--------------------------: | :-----: | :---------: |
+|    ✅     | RPC_URL | RPC url for Ethereum/Polygon |         |             |
```

Comment:
Any chance you could make this dynamic based on the input param? That allows for deploying a single adapter for multiple networks

```suggestion
|    ✅     | [network]_RPC_URL | RPC url for Ethereum/Polygon |         |             |
```

Reply:
I had a setup similar to what you're describing initially.  An alchemy api url was created based on a chain id passed in via request. One adapter supported all chains. @thodges-gh requested that we move to one adapter instance per network in [this comment](https://github.com/smartcontractkit/external-adapters-js/pull/1085#issuecomment-964153619). 

Which setup would you like us to use?

Reply:
Our approach has changed since Thomas left that comment, and we've now moved all adapters to use network name in the RPC_URL env var instead.